src/functionaltests/java/com/ericsson/ei/utils/HttpRequests.java

```diff
@@ -0,0 +1,146 @@
+package com.ericsson.ei.utils;
+
+import com.ericsson.ei.controller.model.SubscriptionResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.FileUtils;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.*;
+
+
+public class HttpRequests {
```

Comment:
Split this in two classes one for get an one for post and use the builder pattern with default field values to create instances. Some time we might need to overwrite the content type which today is hard coded.